test/python/inspectors/test_pylint_inspector.py

```diff
@@ -46,8 +48,11 @@ def test_file_with_issues(file_name: str, n_issues: int):
 
 def test_parse():
     file_name = 'test.py'
```

Comment:
I think it would be great to use the `file_name` variable in the `output` variable instead of _test.py_ directly